cloud/resource_pulsar_cluster.go

```diff
@@ -84,6 +84,13 @@ func resourcePulsarCluster() *schema.Resource {
 				Description:  descriptions["location"],
 				ValidateFunc: validateNotBlank,
 			},
+			"release_channel": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				Default:      "rapid",
+				Description:  descriptions["release_channel"],
+				ValidateFunc: validateNotBlank,
```

Comment:
Maybe you can add a function `validateReleaseChannel` to verify if it is lts, rapid or null.

Reply:
fixed, thanks.